ksql-engine/src/main/java/io/confluent/ksql/function/udf/json/ArrayContainsKudf.java

```diff
@@ -42,10 +42,6 @@
   private static final JsonFactory JSON_FACTORY = new JsonFactory()
```

Comment:
I guess we are satisfied that this `JsonFactory` is thread-safe? Similarly with the `ObjectMapper` below, this is shared across all queries - perhaps having it as an instance variable would be better?

Reply:
Yep, checked the docs, these are thread-safe.